examples/tv-app/android/App/platform-app/src/main/java/com/matter/tv/server/service/ContentAppAgentService.java

```diff
@@ -36,7 +36,7 @@ public class ContentAppAgentService extends Service {
   public static final int FAILED_UNKNOWN = 0x01;
   public static final int FAILED_TIMEOUT = 0x94;
 
-  private static final int COMMAND_TIMEOUT = 8; // seconds
+  private static final int COMMAND_TIMEOUT = 4; // seconds
```

Comment:
was this change intentional? Can you provide a comment explaining why 4 seconds and not 8?

Reply:
if we are touching this, I recommend adding units of measurements for everything that is a unitless integer types. This should be `COMMAND_TIMEOUT_SEC` and below should be `ATTRIBUTE_TIMEOUT_SEC` or you should convert these to a Duration type.

Reply:
@chrisdecenzo  the client  send UDC message to the platform App in main thread，So I lowered the value to avoid ANR

Reply:
@andy31415 yes，we did do [that](https://github.com/project-chip/connectedhomeip/blob/11ea4316f3e5a60f8d34fcad8deeb154803498a8/examples/tv-app/android/App/platform-app/src/main/java/com/matter/tv/server/service/ContentAppAgentService.java#L151)